lib/index.js

```diff
@@ -1,21 +1,50 @@
 var AWS = require('aws-sdk');
 AWS.config.region = process.env.AWS_REGION;
+
+
+
+function set_http_proxy() {
+    var proxySet = false;
+    var proxy = require('proxy-agent');
```

Comment:
do we _need_ a "Proxy Agent" to send email?
https://www.npmjs.com/package/proxy-agent